README.rst

```diff
@@ -0,0 +1,90 @@
+=============================================
```

Comment:
should we have either rst or md so it's not duplicated? I'm guessing you had to add one for pypi